static/js/shared.js

```diff
@@ -153,7 +140,24 @@ window.onerror = function(error, url, line) {
     }
 };
 
-function warn(msg){
-    document.getElementById("alert-modal").style.display="block";
```

Comment:
is warn used anywhere else?

Reply:
nope we added it just for this I thought, but let me check and make sure

Reply:
nevermind it is used for the plugins to warn when there is no GUI component, so I'll split out our error listing to a new function.